Automattic/woocommerce-payments
# IPP HTML Receipts are missing fees and shipping

Fixes #3984 


#### Changes proposed in this Pull Request
- Add fees and shipping information to receipts. 
- Return the receipt HTML on `html_content` key 








#### Testing instructions
 - Connect your local WC Pay client and server.
 - Create an order with Cash on Delivery as payment method. Please use the proxied url of your local site.
 - Start listening for webhooks in your local WC Pay server. You'll need this to refresh your order status after next step.
 - Use the WooCommerce Mobile App to pay the order. You can follow this great guide to set it up locally: pdjVC1-9M-p2
 - Once the payment is suceeded, you'll need to note the payment id reference i.e: you can find it in the order details
 - Using Postman (note that you'll need to set up Authentication as explained in this guide: PbTz5e-12J-p2) make a GET request to /wp-json/wc/v3/payments/readers/receipts/<payment_id>.
 - You should be able to check the HTML response within Postman. The HTML response should be on `html_content` key






-------------------


**Post merge**
